src/app/main.js

```diff
@@ -16,6 +16,13 @@ import VueClipboard from 'vue-clipboard2'
 import '../metrics'
```

Comment:
Email an github link should be centered.
![image](https://user-images.githubusercontent.com/5394573/70320421-b8bdc800-1835-11ea-898e-5ab4624d51ae.png)